src/include/rbd/librbd.h

```diff
@@ -144,6 +144,8 @@ enum {
   RBD_IMAGE_OPTION_JOURNAL_ORDER = 5,
   RBD_IMAGE_OPTION_JOURNAL_SPLAY_WIDTH = 6,
   RBD_IMAGE_OPTION_JOURNAL_POOL = 7,
+  RBD_IMAGE_OPTION_FEATURES_SET = 8,
```

Comment:
@dillaman  Do you mean pass FEATURES_MASK = RBD_FEATURES_SINGLE_CLIENT. and then we can mask it out in librbd, right?

Yes, the RBD_IMAGE_OPTION_FEATURES_CLEAR is working similarly.
```
+ uint64_t features_clear = 0;
+ uint64_t features_set = 0;
+ opts.get(RBD_IMAGE_OPTION_FEATURES_CLEAR, &features_clear);
+ opts.get(RBD_IMAGE_OPTION_FEATURES_SET, &features_set);
+
+ uint64_t conflict = features_clear & features_set;
+ features_clear &= ~conflict;
+ features_set &= ~conflict;
+
+ features |= features_set;
+ features &= ~features_clear;
```